bluemira/equilibria/coils/_grouping.py

```diff
@@ -343,7 +357,8 @@ def from_group_vecs(cls, eqdsk: EQDSKInterface) -> CoilGroup:
                     current=eqdsk.Ic[i],
                     dx=dx,
                     dz=dz,
-                    ctype="PF",
+                    ctype=ct or CoilType.PF,
```

Comment:
Do we want to allow other ctype's here other than PF?

Reply:
yes dummy coils now or any future coil type could be named here.